src/surface/marching-cubes.ts

```diff
@@ -744,8 +751,8 @@ function MarchingCubes (this: MarchingCubes, field: number[], nx: number, ny: nu
         }
       } else {
         // FIXME normals flipping (see above) and vertex order reversal
```

Comment:
This looks great, thanks Fred! I think you could delete this FIXME comment with this fix.

Reply:
Done! Thanks